5.bindings/README.md

```diff
@@ -99,16 +99,15 @@ docker-compose -f ./docker-compose-single-kafka.yml down
 
 ### Setting up a Kafka in Kubernetes
 
-1. Make sure that you installed helm tiller on your cluster
```

Comment:
Helm tiller removed in version 3

Reply:
https://helm.sh/docs/faq/#changes-since-helm-2

Reply:
By default in the `Prerequisites to Run in Kubernetes` link, Helm 3 is the specified version